action.yml

```
@@ -149,7 +149,10 @@ runs:
         steps.pre-installed-python.outputs.python-path == ''
         && steps.new-python.outputs.python-path
         || steps.pre-installed-python.outputs.python-path
-      }} '${{ github.action_path }}/create-docker-action.py'
+      }} '$GITHUB_ACTION_PATH/create-docker-action.py'
```

For shell interpolation to work, the quotes must be double.
```suggestion
      }} "${GITHUB_ACTION_PATH}"/create-docker-action.py
```